src/components/ConfirmationDialog.js

```diff
@@ -75,6 +81,28 @@ export const ConfirmationDialog = hh(class ConfirmationDialog extends Component
 
             div({ id: "lbl_dialogContent", className: "dialog-content" }, [
               this.props.children,
+              div({ 
+                isRendered: this.props.dacSelection !== undefined, 
+                className: "form-group" }, [
+                label({
+                  id: "lbl_selectDAC",
+                  className: "col-lg-12 col-md-12 col-sm-12 col-xs-12 control-label " + this.props.color + "-color", style: customStyles.labelMargin,
+                }, ["Select a DAC to vote on this election"]),
+                  h(AsyncSelect, {
+                    id: "sel_dac",
+                    // key: this.state.dacName,
+                    isMulti: false,
+                    // loadOptions: (query, callback) => this.searchDacs(query, callback),
+                    // onChange: (option) => this.onDacsChange(option),
+                    // value: this.state.dacName,
+                    // noOptionsMessage: () => this.state.optionMessage,
+                    // loadingMessage: () => this.state.optionMessage,
+                    classNamePrefix: "select",
+                    placeholder: "Select DAC...",
+                    className: 'select-autocomplete',
+                    required: true
+                  })
+              ]),
```

Comment:
I see what you mean here. There is a bunch of extra content here that is conditionally rendered. Instead of defining that here, I think it would make sense if the component accepted any number of child divs. In all cases where we call the component like we did before, nothing would be rendered. In the case where we call this dialog from this new location, that is where we would specify the div as another thing to be rendered. Will look for some examples that might illustrate the point.

Reply:
I tested this locally and that entire div (minus the render) can go in AdminManageDul right before the `div({ className: "dialog-description" }` line which is 564 in this PR.

Reply:
Actually, there is a slight difference between having the div in one place vs the other. Not sure if my suggestion will meet the requirement, but it does allow for a cleaner confirmation dialog

Reply:
yes, we can put the select in there, whatever we put below 564 will be rendered as "child" in the dialog. Didn't do that in the first place cause I wasn't sure how the field will be validated if empty from the dialog.

Reply:
@rushtong please take a look to see if this makes more sense now